aws_s3_bucket.tf

```diff
@@ -121,3 +149,254 @@ resource "aws_s3_bucket" "s3MFADeleteOnly" {
     mfa_delete = true
   }
 }
+
+resource "aws_s3_bucket_policy" "s3VersioningMfaFalsePolicy" {
+  bucket = "${aws_s3_bucket.s3VersioningMfaFalse.id}"
+
+  policy = <<POLICY
+{
+  "Version": "2012-10-17",
+  "Statement": [
+    {
+      "Sid": "s3VersioningMfaFalse-restrict-access-to-users-or-roles",
```

Comment:
**Accurics remediation**
 
**[HIGH]: S3 bucket should not be world-writable**
 
**Vulnerability**
Misconfigured S3 buckets can leak private information to the entire internet or allow unauthorized data tampering / deletion
 
**Remediation**
In the S3 console, select the Permissions tab, and then Bucket Policy. Remove policies for s3:List* actions for any principals '*'. If necessary, modify the policy instead, to limit the access to specific principals. Then, select Access Control List, and remove any grants to List objects to Public (everyone) users. Refer to: https://docs.aws.amazon.com/AmazonS3/latest/dev/using-iam-policies.html

source: accurics-